portfolio/src/main/webapp/script.js

```diff
@@ -71,30 +71,40 @@ function fetchComments() {
 
 window.onload = (event) => {
   fetchComments();
-  toggleCommentBox();
+  changeCommentFormDisplay
 };
 
 function onSignIn(googleUser) {
-  const profile = googleUser.getBasicProfile();
-  toggleCommentBox(); 
+  changeCommentFormDisplay(); 
 }
 
 function signOut() {
   const auth2 = gapi.auth2.getAuthInstance();
   auth2.signOut().then(function() {
-    toggleCommentBox();
+    changeCommentFormDisplay();
   });
 }
 
 /*
 * Hides comment when not signed in and shows comments form when
 * signed in with your google account
 */
-function toggleCommentBox() {
+function changeCommentFormDisplay() {
   const auth2 = gapi.auth2.getAuthInstance();
   if (auth2.isSignedIn.get()) {
     document.getElementById("comment-form").style.display = "block";
   } else {
     document.getElementById("comment-form").style.display = "none";
   }
-}
\ No newline at end of file
+}
+
+function commentPostRequest() {
+  const id_token = gapi.auth2.getAuthInstance().currentUser.get().getAuthResponse().id_token;
+  const textbox = document.getElementById("text_input");
+  const xhr = new XMLHttpRequest();
```

Comment:
Might want to stick with the fetch API, it's more modern and I think the tutorials had you all using it.
https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API